ci-operator/step-registry/telcov10n/slcm/tests/telcov10n-slcm-tests-ref.yaml

```diff
@@ -0,0 +1,46 @@
+ref:
+  as: telcov10n-slcm-tests
+  from_image:
+    namespace: ci
+    name: telco-bastion
+    tag: latest
+  commands: telcov10n-slcm-tests-commands.sh
+  timeout: 4h
+  resources:
+    requests:
+      cpu: 100m
+      memory: 200Mi
+  env:
+  - name: ANSIBLE_SKIP_TAGS
+  - name: CNF_IMAGE
+  - name: DCI_PIPELINE_FILES
+  - name: DEBUG_MODE
+  - name: DU_PROFILE
+  - name: ECO_GOTESTS_CONTAINER
+  - name: EDU_CLUSTER
+  - name: EDU_PTP
+  - name: GITLAB_BRANCH
+  - name: OCP_VERSION
+  - name: RUN_EDU_TESTS
+  - name: SITE_NAME
+  - name: SPIRENT_PORT
+  - name: STAMP
+  - name: TEST_ID
+  credentials:
+  - namespace: test-credentials
+    name: slcm-stage-01-ci-project-id
```

Comment:
Is this secret collection shared? if not we can use the `telcov10n` collection.

Reply:
Carlos C. has access but I dunno how to add you and Shai, so OK I think I'm going to migrate.

Reply:
If @ccardenosa can access it that's ok. I can show you how to add others if needed